crypto/x509/x_crl.c

```diff
@@ -429,10 +430,12 @@ X509_CRL_METHOD *X509_CRL_METHOD_new(int (*crl_init) (X509_CRL *crl),
                                      int (*crl_verify) (X509_CRL *crl,
                                                         EVP_PKEY *pk))
 {
-    X509_CRL_METHOD *m;
-    m = OPENSSL_malloc(sizeof(*m));
-    if (m == NULL)
+    X509_CRL_METHOD *m = OPENSSL_zalloc(sizeof(*m));
```

Comment:
is there a reason to zeroing m here ?

Reply:
To be sure to clean all the fields, even if here; all fields are assigned just after.